AnalyticsTests/AnalyticsTests.swift

```diff
@@ -121,19 +121,18 @@ class AnalyticsTests: QuickSpec {
       let integration = analytics.test_integrationsManager()?.test_segmentIntegration()
       expect(integration).notTo(beNil())
       
-      var sent = 0
-
       analytics.flush()
-      integration?.test_dispatchBackground {
-        if let count = integration?.test_queue()?.count {
-          sent = count
-        }
-        else {
-          sent = -1
+      waitUntil(timeout: 60) {done in
```

Comment:
is this the timeout to check for the messages in webhook? if so, you may want to increase it to 3 minutes as 1 minute is sometimes not enough to find them in webhook (3 mins is what library-e2e-tester currently does).

Reply:
It isn't this one, but I pushed a new commit to increase the webhook timeout to 3 minutes - thanks!